osm2lanes/src/placement.rs

```diff
@@ -87,7 +87,7 @@ impl Placement {
     }
 }
 
-#[cfg(tests)]
+#[cfg(test)]
```

Comment:
Eep, good catch!

Reply:
Haha, not even my catch. Just clippy complaining.